<launch>
    <node pkg="dogsim" type="get_path_server" name="get_path_server">
      <param name="path_type" value="rectangle"/>
  </node>
</launch>
